examples/with-nx/apps/app/pages/index.module.css

```diff
@@ -0,0 +1,2 @@
+.page {
```

Comment:
Is this meant to be empty as a placeholder?

Reply:
This was definitely leftover from the generator. Removed 👍🏼